script/demoScripts/demoPolymer.sh

```
@@ -0,0 +1,271 @@
+    RETRY_COUNT=$((RETRY_COUNT + 1))
+  fi
+done
+
+if [ $RETRY_COUNT -eq $MAX_RETRIES ]; then
+  echo "✗ Timeout: CCTP transfer did not complete within the expected time"
+  exit 1
+fi
+
+# query and log usdc balances after transaction
+echo "==================== balances after transaction ====================="
+from_balance_after=$(cast call "$from_token" "balanceof(address)(uint256)" "$user_address" -r "$from_rpc_url" 2>&1) || from_balance_after="<query failed>"
```

Shellcheck (0.11.0)

[warning] from_token is referenced but not assigned (did you mean 'FROM_TOKEN'?).

(SC2154)